### ISSUE TITLE
add missing network

### ISSUE THREAD

**Issue by butlerx** [redbrick/docs-old#5]
- [ ] higgs
- [ ] hadron
- [ ] junos

**Comment by VoyTechnology** [comment]
bind9 moved to services in #12. I'm removing that entry

**Comment by theycallmemac** [comment]
Any of the above exist anywhere now?

**Comment by butlerx** [comment]
i believe these switches still exist @m1cr0man or ylmcc should be able to give more ifno

**Comment by theycallmemac** [comment]
Yes i think they've been since "renamed" - I'll try found out which is which

**Comment by ylmcc** [comment]
Hadron is still around! Higgs is MIA but we now have Steve and Sebastian...
